modules/gcp-gke/modules/gcp-gke-node-pool/node-pool.tf

```diff
@@ -1,6 +1,4 @@
 resource "google_container_node_pool" "node_pool" {
```

Comment:
The resources here and in all the other files now need to explicitly take a project ID either for compute, or the VPC project.